### DISCUSSION TITLE
Migrate Postgresql from dev to prd server

### DISCUSSION THREAD

**Discussion by bestrouting** [nautobot/nautobot#6115] [Q&A]
Hello all,

After fully tested in a lab env, I'd like to backup the database from the dev natuobot and restore the database to a new prod nautobot server.
The reason I want to do that is that the dev nautobot server has a lot of data for the production networks so I don't want to put it again to the  prod nautobot server. I already spent a few weeks for data input in the dev server.
What I am thinking of is the following based on "Network Automation with Nautobot".

1. Run "nautobot-server dumpdata --exclude auth.permission --format json --indent 2 --traceback > nautobot_dump.json" in the dev server
2. Copy the "nautobot_dump.json" to the prod server
   Note) The versions of Nautobot server and Nautobot Apps are already identical between the dev and the prod server.
3. Run "nautobot-server loaddata --traceback nautobot_dump.json" in the prod server.

Can anyone confirm this is right procedures or can you advise any better way to achieve what I want to do?

Thank you in advance.

**Comment by glennmatthews** [comment]
If the prod server is a new install, you might find it simpler to use `pg_dump` and `pg_restore` instead of `nautobot-server dumpdata` and `nautobot-server loaddata`.
